src/components/message-box/index.tsx

```diff
@@ -42,6 +42,8 @@ export const MessageBox = ({
   const overrideMode = useStatusThemeColor(type);
   const aria = modeToAria(type);
 
+  if (message === '') return null;
```

Comment:
Can message be undefined here? Or null? Maybe this is a broader check to use

```suggestion
  if (!message) return null;
```

Reply:
The type asserts that it can't be undefined or null, doesn't it?

Reply:
Yes, the type should handle undefined and null values: 

```tsx
export type MessageBoxProps = {
  ...
  title?: string;
  message: string;
  noStatusIcon?: boolean;
  ...
};

```